typescriptFixme.js

```diff
@@ -0,0 +1,52 @@
+// add the type annotation to explicitly specify the type of the variable:
+var isDone = false;
+var lines = 42;
+var username = "Anders";
+// annotate the type as when it's not known
+var unknownTypeVariable;
+//annotate this array
+var list = [1, 2, 3];
+// annotate the return type of this function
+function littleHorribleAlert() {
+    alert("I am a little annoying box!!");
+}
+var f1 = function (i) {
+    return i * i;
+};
+//change f4 to accept a string or a number as a parameter
+function f2(i) {
+    console.log("The value was " + i);
+}
+var person1 = { name: "Bobby", age: 35, move: function () { } };
```

Comment:
Please use an interface for declaring type person with keys name, age, and move.